Formula/p/phrase-cli.rb

```diff
@@ -1,8 +1,8 @@
 class PhraseCli < Formula
   desc "Tool to interact with the Phrase API"
   homepage "https://phrase.com/"
-  url "https://github.com/phrase/phrase-cli/archive/refs/tags/2.40.0.tar.gz"
-  sha256 "1f2000c179595ea5e8c86ae4e8d20f07d0da37e320dddaa96c8c397faba0862a"
+  url "https://github.com/phrase/phrase-cli/archive/refs/tags/2.42.0.tar.gz"
+  sha256 "2b8a67e269d774b069420c36c3db58eac61d70387aab20b16dca3efc3e8926f9"
```

Comment:
sha256 hash changed
```suggestion
  sha256 "42dd084ef2b622e33c524dabfcf6d7e1f74f52d3a1930e942cc91e25720e7779"
```

Reply:
[ammended](https://github.com/Homebrew/homebrew-core/compare/0279febd8fdc9a1dca929c6deefecc49be18ab77..9b7c689287c03186d8b4434a8176fd1bb12fd3db) in [`9b7c689` (#224970)](https://github.com/Homebrew/homebrew-core/pull/224970/commits/9b7c689287c03186d8b4434a8176fd1bb12fd3db)